sapi/fpm/tests/tester.inc

```diff
@@ -803,6 +833,36 @@ class Tester
         return $this->makeFile($extension, $this->processTemplate($mainTemplate));
     }
 
+    /**
+     * Create pool config templates.
+     *
+     * @param array $configTemplates
+     * @return array
+     * @throws \Exception
+     */
+    private function createPoolConfigs(array $configTemplates)
+    {
+        if (!isset($configTemplates['poolTemplate'])) {
+            unset($configTemplates['main']);
```

Comment:
Why do we need to unset `main` here?

Reply:
This is because `main` is a template for the main pool which adds the includes (it should contain the master process configuration) and this function is supposed to return config for pools only - they are in in the `key (pool name) => value (pool template)` format in this case.